inspirehep/modules/workflows/config.py

```diff
@@ -0,0 +1,179 @@
+                    'path': 'reference.report_numbers',
+                    'search_path': 'report_numbers.value.fuzzy',
+                    'type': 'exact',
+                },
+                {
+                    'paths': [
+                        'reference.publication_info.journal_title',
+                        'reference.publication_info.journal_volume',
+                        'reference.publication_info.artid',
+                    ],
+                    'search_paths': [
+                        'publication_info.journal_title.raw',
+                        'publication_info.journal_volume',
+                        'publication_info.artid',
+                    ],
+                    'type': 'nested',
+                },
+                {
+                    'paths': [
+                        'reference.publication_info.journal_title',
+                        'reference.publication_info.journal_volume',
+                        'reference.publication_info.page_start',
+                    ],
+                    'search_paths': [
+                        'publication_info.journal_title.raw',
+                        'publication_info.journal_volume',
+                        'publication_info.page_start',
+                    ],
+                    'type': 'nested',
+                },
+            ],
+        },
+    ],
+    'doc_type': 'hep',
+    'index': 'records-hep',
+    'source': [
+        'control_number',
+    ],
+}
+"""Configuration for matching all HEP records."""
+
+WORKFLOWS_REFERENCE_MATCHER_JHEP_AND_JCAP_CONFIG = {
+    'algorithm': [
+        {
+            'queries': [
+                {
+                    'path': 'reference.arxiv_eprint',
+                    'search_path': 'arxiv_eprints.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.dois',
+                    'search_path': 'dois.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.isbn',
+                    'search_path': 'isbns.value.raw',
+                    'type': 'exact',
+                },
+                {
+                    'path': 'reference.report_numbers',
+                    'search_path': 'report_numbers.value.fuzzy',
+                    'type': 'exact',
+                },
+                {
+                    'paths': [
+                        'reference.publication_info.journal_title',
+                        'reference.publication_info.journal_volume',
+                        'reference.publication_info.year',
+                        'reference.publication_info.artid',
+                    ],
+                    'search_paths': [
+                        'publication_info.journal_title.raw',
+                        'publication_info.journal_volume',
+                        'publication_info.year',
+                        'publication_info.artid',
```

Comment:
Same consideration as https://github.com/inspirehep/inspire-next/pull/3351/files#r182354877.

Reply:
Same also for my comment.